tools/test_modules/m26700.pm

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env perl
+
+##
+## Author......: See docs/credits.txt
+## License.....: MIT
+##
+
+use strict;
+use warnings;
+
+use Crypt::PBKDF2;
+use Crypt::CBC;
+use Crypt::Mode::CBC;
+
+sub module_constraints { [[0, 256], [32, 32], [-1, -1], [-1, -1], [-1, -1]] }
+
+sub module_generate_hash
+{
+  my $word = shift;
+  my $salt_str = shift // random_hex_string (32);
```

Comment:
The // part is not required. It will be filled with random hex string of the length given in the module_contraints. So I'd set the contraint to 64, 64.